corehq/apps/commtrack/tests/test_programs.py

```diff
@@ -63,3 +62,4 @@ def test_delete(self):
             self.default_program._id,
             SQLProduct.objects.get(product_id=self.products[0]._id).program_id
         )
+        self.domain.delete()
```

Comment:
It's better to use `self.addCleanup(self.domain.delete)` so that even if this test errors out it will still be deleted